astroquery/gaia/core.py

```diff
@@ -191,6 +191,9 @@ def load_data(self, ids, *, data_release=None, data_structure='INDIVIDUAL',
             retrieval type identifier. For GAIA DR2 possible values are ['EPOCH_PHOTOMETRY']
             For GAIA DR3, possible values are ['EPOC_PHOTOMETRY', 'RVS', 'XP_CONTINUOUS', 'XP_SAMPLED',
             'MCMC_GSPPHOT' or 'MCMC_MSC']
+            For GAIA DR4, possible values are ['EPOC_PHOTOMETRY', 'RVS', 'XP_CONTINUOUS', 'XP_SAMPLED',
```

Comment:
Maybe tweak the language here for now. I suppose once DR4 is out, we'll have follow-up PRs that will set that to be the default, so this can be changed to present tense then.

```suggestion
            For GAIA DR4, possible values will be ['EPOC_PHOTOMETRY', 'RVS', 'XP_CONTINUOUS', 'XP_SAMPLED',
```